selfservice/strategy/oidc/provider_generic_test.go

```diff
@@ -12,16 +12,21 @@ import (
 	"github.com/ory/kratos/x"
 )
 
+func makeOIDCClaims() string {
+	return "{\"id_token\":{\"email\":{\"essential\":true},\"email_verified\":{\"essential\":true}}}"
```

Comment:
It would be less error prone and better readable if you encode a `map[string]interface{]` here instead of having a json blob. Something like:
```suggestion
func makeOIDCClaims(t *testing.T) string {
    claims, err := json.Marshal(map[string]interface{}{
        "id_token": map[string]interface{}{...},
    }
	return claims
```

Reply:
Added in new commit